src/index.js

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+// import ReactDOM from 'react-dom/client';
+import { createRoot } from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
 import './index.css';
+import { Provider } from 'react-redux';
+import store from './redux/store';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
-
-const root = ReactDOM.createRoot(document.getElementById('root'));
+// import reportWebVitals from './reportWebVitals';
```

Comment:
- [ ] @misikir21 commented code render your codebase unclean. It is advisable  not to push them to the repo. It's ugly and not professional. I think let's get rid of them :+1: